README.md

```diff
@@ -0,0 +1,14 @@
+# refactoring-workshop-demo
+
+[![JB Research](https://jb.gg/badges/research-flat-square.svg)](https://research.jetbrains.org/)
+
+The project aims to demonstrate how the IntelliJ Platform could be useful for researchers on the example of several
```

Comment:
The project aims to demonstrate how the IntelliJ Platform could be useful for researchers in several ddifferent use cases.